src/plugins/create_global_test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+kdb set system/elektra/globalplugins
```

Comment:
This should not be necessary, see impl. of system/elektra/mountpoints how to do it.